src/documentation/pages/components/link/examples/_no-hover-underline.vue

```diff
@@ -0,0 +1,47 @@
+<template lang="pug">
+  <ComponentExample title="No Underline on Hover" id="noHoverUnderline" :tabs="exampleTabs" additionalClasses="-text">
```

Comment:
i'd recommend using the html convention of using dashes - for ids like in your other examples, e.g. no-hover-underline